lldb/tools/lldb-dap/Handler/VariablesRequestHandler.cpp

```diff
@@ -204,14 +133,13 @@ void VariablesRequestHandler::operator()(
   } else {
     // We are expanding a variable that has children, so we will return its
     // children.
-    lldb::SBValue variable = dap.variables.GetVariable(variablesReference);
+    lldb::SBValue variable = dap.variables.GetVariable(var_ref);
     if (variable.IsValid()) {
       auto addChild = [&](lldb::SBValue child,
                           std::optional<std::string> custom_name = {}) {
         if (!child.IsValid())
           return;
-        bool is_permanent =
-            dap.variables.IsPermanentVariableReference(variablesReference);
+        bool is_permanent = dap.variables.IsPermanentVariableReference(var_ref);
         int64_t var_ref = dap.variables.InsertVariable(child, is_permanent);
```

Comment:
This is now shadowing the new `var_ref`, presumably that's not intentional? Maybe call the new one `variable_ref` or `arg_var_ref`?

Reply:
I changed the name to `child_var_ref`, since this is for a child variable.